### PR TITLE
Auto update of layer version file

### PR DESCRIPTION
**Pull request by kubawach**

### CODE REVIEW

File: splunk-apm/README.md [hunk 1]
```diff
@@ -1,22 +1,24 @@
-# Splunk distribution of OpenTelemetry Lambda
+# Splunk APM layer for AWS lambda
 
-Splunk OTeL lambda is a downstream distribution of opentelemetry-lambda. Supported OpenTelemetry Lambda layers are preconfigured to use Splunk as the tracing backend (direct ingest or SmartAgent). Users can enhance their existing Lambda functions by adding Splunk-managed layer directly. Layer ARNs [are published here](https://github.com/signalfx/lambda-layer-versions/).
+Splunk APM layer for AWS lambda contains various components enabling seamless integration of lambdas with metrics and traces. The components come preconfigured to use Splunk as the backend (direct ingest or SmartAgent). Users can enhance their existing Lambda functions by adding Splunk-managed layer directly. Layer ARNs [are published here](./splunk-apm.md).
```
Comment (laurit) [inline splunk-apm/README.md:3]: Instead of SmartAgent perhaps this should mention splunk opentelemetry connector

**Review by laurit — APPROVED** [review]

File: splunk-apm/README.md (hunk 1 above)
Comment (theletterf) [reply splunk-apm/README.md:3]: Hmmm, why are we hiding OTel here?

File: splunk-apm/README.md [hunk 2]
```diff
@@ -26,28 +28,37 @@ All wrapper layers are preconfigured to use Splunk as the tracing backend.
...
+4. Logging
     
     Logging is controller with `OTEL_LAMBDA_LOG_LEVEL` environment variable. Default set to `WARN`. When set to `DEBUG`, the logging exporter is added to traces exporter in order to help verify exported data.
     
-    > Enabling `DEBUG` generates additional logs, which may in turn increase AWS CloudWatch costs. 
+    > Enabling `DEBUG` generates additional logs, which may in turn increase AWS CloudWatch costs.
+
+5. OpenTelemetry Metrics export
+    
+    By default, OTEL metrics are disabled.
```
Comment (theletterf) [inline splunk-apm/README.md:52]:
```suggestion
    By default, OTel metrics are disabled.
```

File: splunk-apm/README.md [hunk 3]
```diff
@@ -26,28 +28,37 @@ All wrapper layers are preconfigured to use Splunk as the tracing backend.
...
+
+6. SignalFx Metrics export
 
+   By default, SignalFx metrics are enabled with Splunk Metrics extension. Following configuration environment variables are required:
```
Comment (theletterf) [inline splunk-apm/README.md:56]:
```suggestion
   By default, metrics are enabled with the Splunk Metrics extension. Following configuration environment variables are required:
```

File: splunk-apm/README.md [hunk 4]
```diff
@@ -59,21 +70,31 @@ No code change is needed to instrument a lambda. However, the `AWS_LAMBDA_EXEC_W
...
+
+Full documentation with examples can be found [here](https://github.com/signalfx/splunk-extension-wrapper/tree/main/docs).
+
+
+
+### Configuration example
 
 The example assumes the following:
 
+- Both SFx metrics and OTeL traces.
```
Comment (theletterf) [inline splunk-apm/README.md:85]:
```suggestion
- Both Splunk metrics and OTeL traces.
```

(Why SFx?)

**Review by theletterf — CHANGES_REQUESTED** [review]

File: splunk-apm/README.md (hunk 3 above)
Comment (kubawach) [reply splunk-apm/README.md:56]: Is there anything like "Splunk metrics" ? Or in other words - did we rebranded SFx to Splunk in all possible contexts? :)

File: splunk-apm/README.md (hunk 4 above)
Comment (kubawach) [reply splunk-apm/README.md:85]: It's SFx tech - the metrics we're pushing here (as OTeL spec for metrics is not ready yet). I can call them "Splunk metrics" but I'm sure i will spark come confusion among current customers.

File: splunk-apm/README.md (hunk 1 above)
Comment (kubawach) [reply splunk-apm/README.md:3]: We don't :D It's just that first layer contained only OTeL-based tracing wrapper while now it's also SFx (Splunk) metrics extension. So Splunk APM layer in general is more that OTeL.

File: splunk-apm/README.md (hunk 4 above)
Comment (theletterf) [reply splunk-apm/README.md:85]: I've updated my suggestions to remove product names/brands (unless it's necessary to make a distinction here). SFx is not really something we should be using anymore.

File: splunk-apm/README.md (hunk 3 above)
Comment (theletterf) [reply splunk-apm/README.md:56]: Made the suggestion brand agnostic. To be honest, I'm not 100% sure we've rebranded everything, but I'm pretty sure we will.

**Pull request merged**